helpers/parse_args.go

```diff
@@ -15,8 +15,8 @@ import (
 
 	"github.com/Azure/go-autorest/autorest/azure"
 	"github.com/Azure/go-autorest/autorest/to"
+	"github.com/gobuffalo/envy"
```

Comment:
I don't mind taking a dependency on envy, but I'd also like to refactor to use viper as well.

Reply:
i took that out of this commit to focus on just updating the deps